website/docs/reference/resource-configs/grants.md

```diff
@@ -249,7 +249,37 @@ models:
 
 <div warehouse="Redshift">
 
-* Granting to / revoking from is only fully supported for Redshift users (not groups or roles).
+For Redshift, you need to distinguish between users, roles, and groups. It is important to explicitly grant roles and groups access to your models. You can revoke access as well.
+
+**For users:**
+When granting access to individual users, use the `+grants` syntax in your configuration to add or modify permissions for these users without replacing existing configurations. In the following example, you only need to list the users:
+
+```yaml
+models:
+  my_schema:
+    schema: my_schema
+    description: "My schema"
+    +grants:
+      select: ["gspider"]  # example of user access
+```
+
+**For roles and groups:**
+For roles and groups, the `+grants` syntax is not supported. You need to provide the complete list of roles or groups that require access each time you make a change. Additionally, you must prefix each role or group with 'role' or 'group':
```

Comment:
I don't think that the `+` is what is not supported here, rather, what isn't supported is the [inheritance of grants](https://docs.getdbt.com/reference/resource-configs/grants#grant-config-inheritance) 

```suggestion
Inheritance of grants are not supported for groups and roles in Redshift. Accordingly, for every model, you need to provide the complete list of roles or groups that require access each time you make a change to a model.

Additionally, must prefix each role or group with 'role' or 'group':
```